sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/CosmosClientBuilder.java

```diff
@@ -172,6 +174,24 @@ public CosmosClientBuilder connectionPolicy(ConnectionPolicy connectionPolicy) {
         return this;
     }
 
+    /**
+     * Gets the {@link CosmosKeyCredential} to be used
+     * @return cosmosKeyCredential
+     */
+    public CosmosKeyCredential cosmosKeyCredential() {
```

Comment:
Can we have other types of KeyCredential ?
having Cosmos prefix feels redundant in the builder.
If this is the only credential configuration, then it might be good to call it credential  (both places to set and get)
@srnagar